tools/pkglint/lib/packagejson.ts

```diff
@@ -72,6 +72,7 @@ export class PackageJson {
     this.excludeRules = _forceArray(this.json.pkglint && this.json.pkglint.exclude) || (disabled ? [/^.*$/] : []);
 
     function _forceArray(arg: string | string[] | undefined): RegExp[] | undefined {
+      // eslint-disable-next-line eqeqeq
```

Comment:
Can we configure the `eqeqeq` rule so it allows `== null`? It was possible with `TSLint`, and I'd actually even want to _enforce_ this is used instead of `=== undefined` or `=== undefined || === null`...

Reply:
There is an option to ignore `null`, but it seems to only be an issue with `tools/*`. Again, I'm not sure why TSLint wasn't picking it up.

I'd much prefer replacing those few `==` and `!=` with strict checks.

Reply:
The issue is that the strict check equivalent to `x == null` is *extremely* verbose: `x === null || x === undefined`. I'm not into that much typing, when the expressiveness of the short syntax is so nice.

It was not picked up because the `tslint:recommended` settings have ` "triple-equals": { options: ["allow-null-check"] },` -- which I like.

Reply:
Yeah, I thought it was only used in `tools`. I'll add [the `allow-null` option](https://eslint.org/docs/rules/eqeqeq#allow-null)

I have a bigger issue, ESLint isn't picking up `packages/@aws-sdk/*` errors. I'll look into it tonight.

Reply:
Done